ompi/mca/coll/han/coll_han_component.c

```diff
@@ -95,7 +99,25 @@ static int han_open(void)
               2 * cs->han_auto_tune_n * cs->han_auto_tune_c * cs->han_auto_tune_m, file);
         fclose(file);
     }
-    return OMPI_SUCCESS;
+
+    /*
+     * Get the global coll verbosity: it will be ours
+     */
+    param = mca_base_var_find("ompi", "coll", "base", "verbose");
```

Comment:
get it from the base collective framework instead of looking again for the MCA param.

Reply:
Fixed in 217b881bb37be2f9e8641fb49a45df89bd7fc1e6